general/networking/letsencrypt.md

```diff
@@ -120,27 +120,26 @@ echo "0 0 * * *  root  certbot renew --quiet --no-self-upgrade --post-hook 'syst
 
 This section assumes that Jellyfin is running in a Docker container (on Linux). This section also assumes that you wish to run Let's Encrypt in a Docker container as well. The Linuxserver/letsencrypt Docker container has a built-in nginx webserver to handle the reverse proxy.
 
-First, you need to determine a few things. 
+First, you need to determine a few things.
 
-**MAKE SURE YOU HAVE A CNAME FOR JELLYFIN WITH YOUR DNS PROVIDER BEFORE PROCEEDING**
+1. **MAKE SURE YOU HAVE A CNAME FOR JELLYFIN WITH YOUR DNS PROVIDER BEFORE PROCEEDING**
```

Comment:
Can we add an advisement to perform a `ping FQDN` (I prefer nmap -Pn FQDN since ports are important for this as well)

Reply:
I was just doing the minimal change to make markdownlint happy. How come someone keeps on merging docs that don't comply with markdownlint?

Reply:
That mightve been me earlier... 😅. Dkanada should've fixed it earlier.